osu.Game/Screens/OnlinePlay/Playlists/PlaylistsMatchSettingsOverlay.cs

```diff
@@ -326,13 +307,17 @@ private void apply()
 
                 RoomName.Value = NameField.Text;
                 Availability.Value = AvailabilityPicker.Current.Value;
-                Type.Value = TypePicker.Current.Value;
 
                 if (int.TryParse(MaxParticipantsField.Text, out int max))
                     MaxParticipants.Value = max;
                 else
                     MaxParticipants.Value = null;
 
+                if (int.TryParse(MaxAttemptsField.Text, out max))
```

Comment:
Does this not also need an entry in the binds above (at the end of `load()`)? That part handles the transfer from room -> textbox.

Reply:
I guess it would be good to add that, but in the case of playlists you are never adjusting the settings after creation, so this wouldn't be noticeable.